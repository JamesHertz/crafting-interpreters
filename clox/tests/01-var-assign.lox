var x;
print x;

x = 10;
x = x * 2;

print x;
{
    var y = x;
    y = x * 2;
    print y;

    print 2 + (y = 10);
}

print 2 + (x = 0);
