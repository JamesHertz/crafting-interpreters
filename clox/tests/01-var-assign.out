nil
20
40
12
2
